docs/ldml/tr35.html

```diff
@@ -8564,6 +8564,268 @@ <h2><a name="Links_to_Other_Parts" href="#Links_to_Other_Parts"
+		    <li>Of course, only the Variants can contain more than one item: the others are either empty or contain exactly 1 item.</li>
+      </ul>
+	  <h4 >2. Alias elements</h4>
+		  <p>For the languageAlias elements, the <em>type</em> and <em>replacements</em> are languageIds.</p>
+		  <p>For the script-, territory- (aka region), and variant- Alias elements, the type and replacements are interpreted as a languageIds, <em>after</em> prefixing with &ldquo;und-&rdquo;. Thus</p>
+		  <code>&lt;territoryAlias type="AN" replacement="CW SX BQ" reason="deprecated"/&gt;</code>
+		  <p>is interpreted as:</p>
+		  <code>&lt;territoryAlias type="und-AN" replacement="und-CW und-SX und-BQ" reason="deprecated"/&gt;</code>
+		  <p>Note that for the case of territoryAlias, there may be multiple replacement values separated by spaces in the text (such as replacement="und-CW und-SX und-BQ"); other rules only ever have a single replacement value.</p>
+		  <p> </p>
+	  <h4 >3. Matches</h4>
+		  <p>A rule matches a source if and only for all fields, each <em>source</em> field ⊇ <em>type</em> field.</p>
+		  <blockquote>
+		  <p><em>Examples:</em></p>
+		  <p>source=&ldquo;ja-heploc-hepburn&rdquo; and type=&rdquo;und-hepburn&rdquo;</p>
+		  <table class='simple'>
+		    <tbody>
+		      <tr>
+		        <td colspan="1" rowspan="1"><p>{ja} ⊇ {} </p></td>
+		        <td colspan="1" rowspan="1"><p>success, und = {}</p></td>
+	          </tr>
+		      <tr>
+		        <td colspan="1" rowspan="1"><p>{hepburn, heploc} ⊇ {hepburn}</p></td>
+		        <td colspan="1" rowspan="1"><p><strong>success</strong></p></td>
+	          </tr>
+	        </tbody>
+		    </table>
+		  <p>so the rule matches the source. (Note that order of variants is immaterial to matching)</p>
+		  <p>&nbsp;</p>
+		  <p> </p>
+		  <p>source=&ldquo;ja-hepburn&rdquo; and type=&rdquo;und-hepburn-heploc&rdquo;</p>
+		  <table class='simple'>
+		    <tbody>
+		      <tr>
+		        <td colspan="1" rowspan="1"><p>{ja} ⊇ {} </p></td>
+		        <td colspan="1" rowspan="1"><p>success, und = {}</p></td>
+	          </tr>
+		      <tr>
+		        <td colspan="1" rowspan="1"><p>{hepburn} ⊉ {hepburn, heploc}</p></td>
+		        <td colspan="1" rowspan="1"><p><strong>failure</strong></p></td>
+	          </tr>
+	        </tbody>
+		    </table>
+		  <p>so the rule does not match the source.</p></blockquote>
+	  <h4 >4. Replacement</h4>
+	  <p>A matching rule can be used to transform the source fields as follows</p>
+		  <ul>
+		    <li>if type.field ≠ {}
+		      <ul>
+		        <li>source.field = (source.field - type.field) ∪ replacement.field</li>
+	          </ul>
+		    </li>
+      
+		    <li>else if source.field = {} and replacement.field ≠ {}
+		      <ul>
+		        <li>source.field = replacement.field</li>
+	          </ul>
+		    </li>
+      </ul>
+		  <p><em>Example:</em></p>
+		  <blockquote><p>source=ja-Latn- fonipa-hepburn-heploc</p>
```

Comment:
Should probably fix the space after ja-Latn-, that could be confusing

Reply:
fixing 2 instances